src/terminal/terminal_handlers.c

```diff
@@ -1373,3 +1377,22 @@ int guac_terminal_ctrl_func(guac_terminal* term, unsigned char c) {
 
 }
 
+/**
+ * xterm does not implement APC functions and neither do we.
+ */
+int guac_terminal_apc(guac_terminal* term, unsigned char c) {
+
+    /* Look for the "ESC \" (string terminator) sequence,
+     * while ignoring other chars. */
+    static char escaping = 0;
```

Comment:
If representing a boolean value, please use the semantically-correct `bool`.

We have to live with using `int` within public APIs like libguac to avoid issues with third-party code that redefines `bool` in a non-standard way, as well as within anything dealing with FreeRDP (an example of such third-party code), but the internal terminal emulator is definitely safe to use `bool`.